src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -380,10 +391,23 @@
                         throw "Tab target " + targetId + " does not exist";
                     }
 
-                    tabs[i].addEventListener('click', function (e) {
-                        select(this);
+                    Sfjs.addEventListener(tabs[i], 'click', function (e) {
+                        var target;
+
+                        if (!e) {
+                            e = window.event;
+                        }
+
+                        if (e.target) {
+                            target = e.target;
+                        }
+                        else if (e.srcElement) {
+                            target = e.srcElement;
+                        }
```

Comment:
this duplicated code to get the target of the event should probably be extracted to an helper function